projects/schematics/src/ng-add/files/auth-config.module.ts

```diff
@@ -0,0 +1,32 @@
+import { APP_INITIALIZER, NgModule } from '@angular/core';
+import { AuthModule, OidcConfigService } from 'angular-auth-oidc-client';
+
+export function configureAuth(oidcConfigService: OidcConfigService): () => Promise<any> {
+    return () =>
+        oidcConfigService.withConfig({
+            stsServer: 'please-enter-sts',
+            redirectUrl: window.location.origin,
+            postLogoutRedirectUri: window.location.origin,
+            clientId: 'please-enter-clientId',
+            scope: 'please-enter-scopes',
```

Comment:
Would a comment help here for the scope configuration?

// 'openid profile '  + your scopes, offline_access if you refresh